app/controllers/projects_controller.rb

```diff
@@ -66,10 +66,20 @@ def update
 
     if params[:project][:rerun] == 'reducers'
       rerun_reducers
-      respond_with project, location: project_path(@project, anchor: 'reducers')
+      respond_with project, location: project_path(project, anchor: 'reducers')
     else
+      was_paused = project.paused?
       project.update(project_params.except('display_name'))
-      respond_with project
+
+      if !was_paused && project.paused?
+        flash[:notice] = 'Pausing project'
+      end
+
+      if was_paused && project.active?
```

Comment:
`is_paused` seems to make more sense if the state is current. Or just use `project.paused`

Reply:
(see above)